doc/api/http.md

```diff
@@ -841,8 +841,25 @@ Note that the socket timeout logic is set up on connection, so
 changing this value only affects *new* connections to the server, not
```

Comment:
I'd rather not change it in this PR, it would be unrelated stylistic change. But yeah, a nice thing to do in a follow-up PR.